styles/duckduckgo/catppuccin.user.css

```diff
@@ -448,6 +451,14 @@
     .module__inner-toggle::after {
       background-color: @surface2 !important;
     }
+    .module--lyrics:not(.is-expanded)
+      .module--lyrics__footer.can-expand::after {
+      background: linear-gradient(rgba(0, 0, 0, 0), @surface0);
```

Comment:
Can we avoid using a non-palette color in this gradient?

Reply:
i used it after checking that there are other non palette colors in the style
![image](https://github.com/user-attachments/assets/a8431922-60bd-4c35-9a78-8779ec0fb9f4)
there are 6 other occurrences of using an rgba for an invisible color
it's an invisible color with 0 opacity so there's no difference between it and `fade(@surface0, 0%)`. also when reading the code `rgba(0, 0, 0, 0)` is clearer and feels more like an invisible color while the other form feels more of a palette color that has an opacity
but that's my opinion, if you really think that using fade with a palette color and zero percentage is more appropriate i'll change it

Reply:
Ah I didn't see it was 0. Why not just use `transparent` for clarity?

Reply:
oh i forgot that i can do that. i replaced all the transparent colors with `transparent`. is it alright now?